cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -41,12 +42,13 @@ func NewCommand() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "the cluster name")
+	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "whether retain the broken nodes for debugging")
```

Comment:
delete probably shouldn't take a retain? since it's always going to delete

Reply:
sure